homeassistant/util/unit_system.py

```diff
@@ -116,15 +122,8 @@ def __init__(
         self.volume_unit = volume
         self.wind_speed_unit = wind_speed
 
-    @property
-    def is_metric(self) -> bool:
-        """Determine if this is the metric unit system."""
-        return self.name == CONF_UNIT_SYSTEM_METRIC
-
-    @property
-    def is_us_customary(self) -> bool:
-        """Determine if this is the US customary unit system."""
-        return self.name == CONF_UNIT_SYSTEM_IMPERIAL
```

Comment:
Imperial != US...
We should fix the actual configuration and constants before starting to mix up things like this. It will keep haunting us as legacy if we don't.

Reply:
Sorry the line was meant for line 208, which only aliases it, not fixing the issue.

Reply:
I will create the new (separate) US_CUSTOMARY_SYSTEM in a follow-up PR, once every component stops using the "name" of the unit system.